app/commands/user/bootstrap.rb

```diff
@@ -0,0 +1,11 @@
+class User
```

Comment:
From V2, we create auth tokens in the Bootstrap service. I decided to do the same here.